### ISSUE TITLE
Can’t change HSM status anymore

### ISSUE THREAD

**Issue by kungktd** [jason0x43/hacs-hubitat#218]
Since a few week's, I can’t change the HSM status from HA automations.
It use to work fine and I haven’t done any changes besides updating HA and HE with official releases.

I can see the following in the trace log in HA,
```
Stopped because an error was encountered at June 18, 2023 at 4:04:06 AM (runtime: 0.09 seconds)
'NoneType' object has no attribute 'lower'
```

**Comment by kungktd** [comment]
I just tried to add the Hub ID in the HE automation and that worked. Strange because it worked without that before…

**Comment by jason0x43** [comment]
It looks like this is a bug that was introduced with the last update. Ironically, the last update made it so users wouldn't have to specify a hub ID to send a command when they had multiple hubs. 😄

**Comment by jason0x43** [comment]
This is fixed in 0.9.7.
